<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CKeyGeneratorWidget</class>
 <widget class="QWidget" name="CKeyGeneratorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CKeyGeneratorWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="rightMargin">
    <number>60</number>
   </property>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="leditKeyEncry">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="labelKeyEncry">
     <property name="text">
      <string>Secret Key:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>AES</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="leditKeySource"/>
   </item>
   <item row="1" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="labelKeySource">
     <property name="text">
      <string>Key Prototype:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pbtnGenerate">
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="labelEncryType">
     <property name="text">
      <string>Encryption Type:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
